models/models.js

```diff
@@ -73,6 +73,25 @@ exports.addNewComment = (commentAdd) => {
     });
 };
 
+exports.incrementArticleVotes = (article_id, inc) => {
+  if (inc === null || inc === undefined) {
+    return Promise.reject({ status: 400, msg: 'Invalid increment vote value' });
+  }
```

Comment:
Id just make sure you're follow TDD - you seem to have code here that doesnt match a test. 
Write the test -> see it fail -> write the code that makes the test pass. You can run into lots of issues if you dont see it pass first

Reply:
changed this line to make sure my test passes
if (isNaN(inc)) {
    return Promise.reject({ status: 400, msg: 'Invalid vote value' });
  }